code/modules/roguetown/roguejobs/cook/tools/milling.dm

```diff
@@ -25,7 +25,10 @@
 /obj/structure/fluff/millstone/proc/grindUp(list/obj/item/to_grind, mob/user)
 	for(var/obj/item/itemtogrind in to_grind)
 		if(do_after(user, 8, target = src))
-			new itemtogrind.mill_result(src.loc)
+			if (!isnull(itemtogrind.mill_result)) // prevent the millstone breaking if no mill_result is defined
+				new itemtogrind.mill_result(src.loc)
+			else
+				new itemtogrind.type(src.loc)
 			to_grind -= itemtogrind
```

Comment:
I don't see anywhere in this file that actually _deletes_ the item that's been ground. This is also a problem. Please add a qdel here for it.